.github/dependabot.yml

```diff
@@ -4,4 +4,5 @@ updates:
   directory: "/"
   schedule:
     interval: weekly
-  open-pull-requests-limit: 5
+  # Disable updates
+  open-pull-requests-limit: 0
```

Comment:
What does this mean?

Reply:
Suppose to disable dependabot for a while 😂.
Will enable again until upgrading rust-toolchain and fix issue https://github.com/scroll-tech/rollup-explorer-backend/issues/107.